mkosi.postinst

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+[ "$1" == "final" ] || exit 0
+
+systemctl enable systemd-networkd
+systemctl enable ironic-python-agent
+systemctl set-default -f multi-user.target
+systemctl disable -f graphical.target
```

Comment:
Please move this steps directly into `mkosi.extra`, as all of those are just symlinks in the file system.

Reply:
The problem is they are symlinks installed according to the internal logic of systemd and depending on the packages installed in the prior step.
More importantly, I cannot remove links with `mkosi.extra`, only add.

Reply:
You can `mask` services to not start by symlinking to `/dev/null`.

Reply:
Also, if `default.target` points as symlink to `multi-user.target`, then the `graphical.target` won't be started (except if explicitly requested).

Reply:
Which shows your knowledge of systemd exceeds mine, and explains why I think `systemctl disable -f graphical.target`  is much more readable and easy to understand, than having a symlink in `mkosi.extra/etc/systemd/...` pointing to `/dev/null`.

In the end, the difference is stylistic, isn't it? I think, neither solution is better in any objective metric.

Reply:
I've removed `systemctl disable -f graphical.target`.